packager/mpd/base/xml/xml_node.cc

```diff
@@ -429,6 +435,16 @@ bool RepresentationXmlNode::AddLiveOnlyInfo(
     if (IsTimelineConstantDuration(segment_infos, start_number)) {
       segment_template.SetIntegerAttribute("duration",
                                            segment_infos.front().duration);
+      if (FLAGS_dash_add_last_segment_number_when_needed) {
+        uint32_t last_segment_number = 0;
+        for (const auto& segment_info_element: segment_infos) {
+	  last_segment_number = last_segment_number + 
+		                segment_info_element.repeat + 1;
+	}
```

Comment:
Two more spaces indent.

Reply:
This is not done yet. You may also remove the brackets for the for loop in line 440.